vmdb/app/models/miq_provision_virt_workflow.rb

```diff
@@ -1385,6 +1367,19 @@ def self.from_ws_ver_1_x(version, userid, template_fields, vm_fields, requester,
     $log.error "#{log_header}: <#{err}>"
     raise err
   end
+
+  private
+
+  def exit_pre_dialog
+    @running_pre_dialog              = false
+    @values[:pre_dialog_vm_tags]     = @values[:vm_tags].dup
+    @values[:forced_sysprep_enabled] = 'fields' if @values[:sysprep_enabled] == 'fields'
+
+    if (sdn = @values[:sysprep_domain_name]).presence.kind_of?(String)
+      @values[:sysprep_domain_name]        = [sdn, sdn]
+      @values[:forced_sysprep_domain_name] = [sdn]
```

Comment:
wow, this wasn't readable before... now it is... why are we storing this value in 3 places?